zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -214,8 +216,7 @@ pub(crate) const CONSENSUS_BRANCH_IDS: &[(NetworkUpgrade, ConsensusBranchId)] =
     (Heartwood, ConsensusBranchId(0xf5b9230b)),
     (Canopy, ConsensusBranchId(0xe9ff75a6)),
     (Nu5, ConsensusBranchId(0xc2d6d0b4)),
-    // TODO: Use the real consensus branch ID once it's specified.
-    (Nu6, ConsensusBranchId(0xdeadc0de)),
+    (Nu6, ConsensusBranchId(0x7fffffff)),
```

Comment:
Note: This will have to be the same as the `zcash_primitives` consensus branch id for nu6 for the acceptance test in this PR to pass.